### ISSUE TITLE
0.9.3.0 Source Build Test

### ISSUE THREAD

**Issue by soup-bowl** [soup-bowl/opensimulator-docker#2]
# Source Build Test

This test is building the OpenSimulator code from source. 0.9.3.0 onwards is using Dotnet Core, which means I've been able to switch to the [Dotnet Runtime container](https://hub.docker.com/_/microsoft-dotnet-runtime) instead of Mono.

[Code](https://github.com/soup-bowl/opensimulator-docker/tree/main/beta/source) | [Container](https://github.com/soup-bowl/opensimulator-docker/pkgs/container/opensimulator-docker/?tag=source)

## For Graduation

Graduation is blocked on this build since this is building edge, and so working on code that is not designed for production use. This is being used for future-proofing the container against the 0.9.3.0 release.

If you build the container from source, ensure to do a full rebuild to get the latest core code.

**Comment by soup-bowl** [comment]
Not sure why, but on a fresh host I get:

> Failed to create CoreCLR, HRESULT: 0x80070008

This is all the information I get...

Edit: running `./OpenSim` directly doesn't experience this issue. Will monitor.

**Comment by soup-bowl** [comment]
Now fixed by commit ad8c3db83e5f1e1673fc9b5b6152744f0c2afbf1. Original `dotnet6` builds now tagged respectively, readme updated.

> Note to say that the fact it is called `source` is misleading. It implies that it is a mainline source build, when this is actually being built from `dotnet6` branch - a divergent branch.
>
> Renaming `source` to `dotnet6` (based on the branch name) might be clearer.
> 
> Edit: On re-inspection, the branch I'm using I presume is the trial of switching to dotnet6, whereas `master` is being built using dotnet (and much further ahead). I perhaps should build both.

**Comment by soup-bowl** [comment]
PR #6 adds Windows support, tied to this version. I'm not sure who will actually _use_ it, but it worked so why not.

**Comment by soup-bowl** [comment]
With #8, 0.9.3.0 is now released and so the adjustments from this branch have been merged into the versioned images. This now just tracks the source build.
